guides/training_with_built_in_methods.py

```diff
@@ -159,8 +159,8 @@
 
 If your model has multiple outputs, you can specify different losses and metrics for
 each output, and you can modulate the contribution of each output to the total loss of
-the model. You will find more details about this in the **Passing data to multi-input,
-multi-output models** section.
+the model. You will find more details about this in the [Passing data to multi-input,
```

Comment:
1. the link should fit on a single line (otherwise it won't render correctly)
2. please replicate the change in the `ipynb` and `md` files

Reply:
Sorry but what do you mean by 'fit on a single line'